pyproject.toml

```diff
@@ -90,6 +92,8 @@ dependencies = [
   "spectres",
   "tqdm",
   "requests",
+  "dense_basis@git+https://github.com/kartheikiyer/dense_basis#egg=9c404ff",
```

Comment:
Do we need the egg, can we not point directly?

Reply:
No, we don't 😂 but we can't yet simplify this to just the PyPi version of `dense_basis` because it hasn't been updated with the now merged changes to the `sklearn` input. I think we just leave that for the future. It's pointing to the main branch for now.